branch-naming.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
 exit_status=0
-currentbranch=`git symbolic-ref --short HEAD`
-if [ $(expr length "$currentbranch") -gt 70 ]
+currentbranch=$(git symbolic-ref --short HEAD)
+dependabot="dependabot"
+if [ "$(expr length "$currentbranch")" -gt 70 ] && [[ ! "$currentbranch" =~ $dependabot ]]
```

Comment:
maybe add a comment explaining why the limit is not enforced on branches created by dependabot

Reply:
Comment added on https://github.com/Kpler/kp-pre-commit-hooks/pull/18/commits/4277dbeb51e8b679a1f70606e993cf507d948f94